samples/client/petstore/typescript-fetch/builds/es6-target/api.ts

```diff
@@ -35,7 +36,7 @@ export const COLLECTION_FORMATS = {
  * @interface FetchAPI
  */
 export interface FetchAPI {
-    (url: string, init?: any): Promise<Response>;
+    (url: string, init?: any): Promise<any>;
```

Comment:
@wing328 @jeff-99 Is this change intentional? It reverts #6774.

Reply:
What I don't understand is: if I run the tests today, without modifying anything, I find that this is changed back to `Response`. So something is broken in the way CI is being done for pull requests, because if the CI was working correctly it would reject this change.

I suspect the fact that there is a merge commit in here somehow confused things. In the future, it is probably best to require all pull-requests to be a single chain of commits that is rebased to the tip of master.

Reply:
What I don't understand is: if I run the tests today, without modifying anything, I find that this is changed back to Response. So something is broken in the way CI is being done for pull requests, because if the CI was working correctly it would reject this change.

I suspect the fact that there is a merge commit in here somehow confused things. In the future, it is probably best to require all pull-requests to be a single chain of commits that is rebased to the tip of master.